02-starting-with-data.Rmd

```diff
@@ -111,10 +111,143 @@ Based on the output of `str(surveys)`, can you answer the following questions?
 ```
 --->
 
-As you can see, many of the columns consist of integers, however, the columns
-`species` and `sex` are of a special class called a `factor`. Before we learn
-more about the `data.frame` class, let's talk about factors. They are very
-useful but not necessarily intuitive, and therefore require some attention.
+## Inspecting `data.frame` Objects
+
+We already saw how the functions `head()` and `str()` can be useful to check the
+content and the structure of a `data.frame`. Here is a non-exhaustive list of
+functions to get a sense of the content/structure of the data.
+
```

Comment:
suggest

'of the data.' -> 'of the data. Let's try them out!'

So that instructors know they should be typing them.